tmail-backend/apps/distributed/src/main/conf/jmap.properties

```diff
@@ -52,4 +52,11 @@ calendarEvent.reply.mailTemplateLocation=file://eml-template/
 # The supported languages for replying to CalendarEvent emails
 calendarEvent.reply.supportedLanguages=en,fr
 
-url.prefix=http://localhost
\ No newline at end of file
+url.prefix=http://localhost
+
+# OIDC configuration
```

Comment:
Let's add `OidcAuthenticationStrategy` in the auth strategies as an example here?